packages/headless/src/app/engine.test.ts

```diff
@@ -16,6 +16,7 @@ describe('engine', () => {
         organizationId: 'orgId',
         accessToken: 'token',
         platformUrl: 'https://www.coveo.com/',
+        name: 'myEngine',
```

Comment:
I would probably make the default something like `coveo-headless` so that a developer using the redux dev tools in a project that itself has redux would clearly see, by default, that this state is owned by headless.

Reply:
For the tests here it's fine 😉  I see you added `coveo-headless` in the default which is good